libs/remix-ui/tabs/src/lib/remix-ui-tabs.tsx

```diff
@@ -135,10 +132,36 @@ export const TabsUI = (props: TabsUIProps) => {
     return () => { tabsElement.current.removeEventListener('wheel', transformScroll) }
   }, [])
 
+  const getExt = (path) => {
+    const root = path.split('#')[0].split('?')[0]
+    const ext = root.indexOf('.') !== -1 ? /[^.]+$/.exec(root) : null
+    if (ext) return ext[0]
+    else return'txt'
+  }
+
   return (
     <div className="remix-ui-tabs d-flex justify-content-between border-0 header nav-tabs" data-id="tabs-component">
       <div className="d-flex flex-row" style={{ maxWidth: 'fit-content', width: '97%' }}>
-        <div className="d-flex flex-row justify-content-center align-items-center m-1 mt-2">
+        <div className="d-flex flex-row justify-content-center align-items-center m-1 mt-1">
+          <OverlayTrigger placement="bottom" overlay={
+            <Tooltip id="overlay-tooltip-run-script">
+              <span>
+                Run script
+              </span>
+            </Tooltip>
+          }>
+            <button
+              className="btn text-success py-0"
+              disabled={!(currentFileExt === 'js' || currentFileExt === 'ts')}
```

Comment:
you don't compile if a `sol` file is selected? that maybe be better actuallly...

Reply:
to compile? I though to do it in a separate PR since it is not included in the issue. but I can do it